player/main.c

```diff
@@ -425,6 +425,16 @@ int mp_initialize(struct MPContext *mpctx, char **options)
     return 0;
 }
 
+#ifdef __SANITIZE_ADDRESS__
+// Suppressions for mpctx->quit_fast
+__attribute__((visibility("default"), weak))
```

Comment:
If this is no longer part of libmpv then the weak attribute can be dropped.

Reply:
Well, `main.c`  is included in libmpv.

Reply:
Never mind then.

Reply:
This could be refactored, but that's topic for another PR.

Reply:
libmpv is quite literally just all of mpv's source files but compiled as a library. It's pretty convenient, so all that to say: please don't change it heh.